app/views/agenda/agenda.html.erb

```diff
@@ -1,26 +1,29 @@
-
-<div class="container mt-5 text-center">
+<div class="simple-calendar">
+ <div class="container mt-5 text-center">
   <h1>Agenda</h1>
+  <%= link_to "Crea tu cita", new_appointment_path, class:"btn btn-outline-secondary btn-sm" %>
   <%= week_calendar(events: @appointments) do |date, appointments| %>
-  <%= date.day %>
+    <%= date.day %>
 
-   <% appointments.each do |appointment| %>
-  <div class="event">
-    <div class = "card-header">
-      <%= link_to appointment.title, appointment_path(appointment) %>
-    </div>
-  </div>
-  
-<div class="card-footer">
-  <p class="card-text">
-  <%= appointment.start_time.strftime("from %I:%M %p")%>
-  </p>
-  <p class="card-text">
-  <%= appointment.end_time.strftime("to %I:%M %p")%>
-  </p>
-</div>
-  
+    <% appointments.each do |appointment| %>
+      <div class="event">
+        <div class = "card-header">
+          <%= link_to appointment.title, appointment_path(appointment) %>
+        </div>
+        <div class="card-body">
+          <p>
+            <%=appointment.description%>
+          </p>
+        </div>
+        <div class="cars-footer">
+          <p class="card-text">
+            <%= appointment.start_time.strftime("%I:%M %p")%>
+            <%= appointment.end_time.strftime("- %I:%M %p")%>
```

Comment:
Maybe?
```suggestion
            <%= appointment.start_time.strftime("%I:%M %p") %> -
            <%= appointment.end_time.strftime("%I:%M %p") %>
```

Reply:
that doesn't work 
<img width="170" alt="Captura de pantalla 2024-05-02 a la(s) 6 20 57 p m" src="https://github.com/NadiaArroyo25/prueba1/assets/130617270/21ada1c0-1a20-4b0f-90b0-e44b3846af01">